compiler/src/iree/compiler/Dialect/LinalgExt/IR/TilingInterfaceImpl.cpp

```diff
@@ -389,6 +389,60 @@ GatherOp::generateResultTileValue(OpBuilder &builder, unsigned resultNumber,
   return getTiledImplementation(builder, offsets, sizes);
 }
 
+LogicalResult GatherOp::generateScalarImplementation(OpBuilder &b, Location loc,
+                                                     ValueRange ivs) {
+  auto indexDepth = getIndexDepth();
+  Value result = b.create<memref::LoadOp>(loc, getOutput(), ivs);
+  SmallVector<Value> starts;
+  SmallVector<Value> loadIndices;
+  append_range(loadIndices, ivs.take_front(getBatchRank()));
+  //
```

Comment:
You are right that a blank line before a commet line usually helps readibility. Please replace `//` with a blank line. For the below, please either group related code in a chunk or add a blank line + a comment for the snippets/variables.

https://google.github.io/styleguide/cppguide.html#Vertical_Whitespace